tests/func/metrics/test_show.py

```diff
@@ -110,3 +110,31 @@ def test_missing_cache(tmp_dir, dvc, run_copy_metrics):
     remove(stage.outs[0].cache_path)
 
     assert dvc.metrics.show() == {"": {"metrics.yaml": 1.1}}
+
```

Comment:
Let's also add a test for showing something that is not metric in other branches.